### ISSUE TITLE
Is there a way to make it work on IPAD?

### ISSUE THREAD

**Issue by armsp** [schollz/croc#500]
In the README it says, it can be used to transfer files between any two computers. I was wondering how to get it to work when one of the computers is an IPAD.

Is there a way or any workaround to make it work between a regular PC where croc is installed and an IPAD?

**Comment by ChienTingLu** [comment]
Maybe you can try apps like iSH or LibTerm? Both can be downloaded from Appstore.

I don't have an iPad at my hand right now. So this is just a guess. (plz reply if it do work XDDD

**Comment by utopiatopia** [comment]
> iSH

iSH is notorious for being ***EXTREMELY*** slow due to emulating x86. Unless you have a new device, expect large file transfers to take 1hr or more. For reference, my iPhone 13 Pro can do 600kb/s. 

> LibTerm

I haven't used this however AFAIK, it wouldn't be advised to use this method. It requires you to compile croc to LLVM IR, which I believe is unstable and/or not easy to do.

> a-shell

I know someone is going to mention this, however this requires you to compile to their special WASM, which I know won't be easy.


TL;DR: While it's technically possible to run it, it's impractical to run.

**Comment by Marksdo** [comment]
I made a simple iOS croc app iCroc.

https://iCroc.marksdo.com or search iCroc in the app store
